utils/build/docker/python/flask/app.py

```diff
@@ -470,6 +471,14 @@ def format_error(errors):
     return {"errors": formatted_errors}
 
 
+@app.route("/add_event")
+def add_event():
+    span = opentelemetry.trace.get_current_span()
+    if span is not None:
+        span.add_event(name="span.event", attributes={"string": "value", "int": 1}, timestamp=datetime.utcnow())
```

Comment:
Instead of hard coding these fields can we pass them in as arguments. This will allow us to re-use this endpoint for other tests